youtube_dl/downloader/dash.py

```diff
@@ -40,7 +40,8 @@ def combine_url(base_url, target_url):
         fragment_retries = self.params.get('fragment_retries', 0)
         skip_unavailable_fragments = self.params.get('skip_unavailable_fragments', True)
 
-        def append_url_to_file(target_url, tmp_filename, segment_name):
+        def append_url_to_file(tmp_filename, segment, skip_unavailable_fragments):
```

Comment:
> when it's ready to be used by all the fragment downloaders.

I feel it ready after this PR. Is there anything to be changed for skipping/retrying?